modules/insights-operator-enable-obfuscation.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc
+
+
+
+[id="insights-operator-enable-obfuscation_{context}"]
+= Enabling Insights Operator data obfuscation
+
+You can enable obfuscation to mask sensitive and identifiable IPv4 addresses and cluster base domains that the Insights Operator sends to link:https://cloud.redhat.com[cloud.redhat.com].
+
+[WARNING]
+====
+Although this feature is available, Red Hat recommends keeping obfuscation disabled for a more effective support experience.
+====
+
+Obfuscation assigns non-identifying values to cluster IPv4 addresses, and uses a translation table that is retained in memory to change IP addresses to their obfuscated versions throughout the Insights Operator archive before uploading the data to link:https://cloud.redhat.com[cloud.redhat.com].
+
+For cluster base domains, obfuscation changes the base domain to a hardcoded substring. For example, `cluster-api.openshift.example.com` becomes `cluster-api.<CLUSTER_BASE_DOMAIN>`.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as `cluster-admin`.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Search for the *support* secret using the *Search by name...* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
```

Comment:
I apologize, I missed this one on my first run through.
s/Search by name.../Search by name

Reply:
Changed it, I wasn't sure because the field has the three dots in it.

Reply:
Yeah, I totally understand where the inclination comes from :) Per style guide: 

> In documentation, do not include punctuation in a label, such as an ellipsis that indicates that more information is required from the user, unless one of the following conditions applies:
> - The omission of the punctuation would result in confusion over the meaning of the interface element.
> - You are using a tool that extracts the wording on the label, including the punctuation, for use in the documentation.